client/client-multi/client-java-core/src/main/java/org/kaaproject/kaa/client/KaaClient.java

```diff
@@ -55,7 +56,7 @@
      * @param record
      *            New log record object
      */
-    void addLogRecord(Log record);
+    BucketFuture addLogRecord(Log record);
```

Comment:
This is wrong. as far as i understand, BucketFuture is generic class. We should return Future<BucketInfo> here.